### DISCUSSION TITLE
MAUI Razor Component CSS not compiling on iOS local device (iPhone)

### DISCUSSION THREAD

**Discussion by PBTests** [dotnet/maui#9475] [General]
**What's the issue?**

Boilerplate .NET MAUI Blazor application does not deploy to iOS physical device correctly. 

The Blazor component CSS appears to not compile, meaning the navigation does not render correctly.

### **Everything renders as expected on the iOS Simulator, Android**, etc...

Latest XCode / VSMac / VS2022 / VS2022-pre / Workloads / Blabla

Tried everything

Seriously... 

Can anyone help?

**Steps to reproduce...**

1. Create a new MAUI Blazor application (VS22/22p)
2. Debug on paired Mac to specified Local Device (iOS 15.5) (effectively deploying to physical device)
3. Result: No classic blazor purple coloured navigation style due to lack of component CSS rendering  

**Screenshot**

![image](https://user-images.githubusercontent.com/56082224/185001233-100248d7-d180-436a-b5da-6ab3fe0b6e74.png)

**Comment by Laftek** [comment]
you should create a new issue to get an answer faster

**Reply by PBTests** [reply]
Done - https://github.com/dotnet/maui/issues/9723

**Comment by GolfJimB** [comment]
Same issue here, app looks great on android but half my elements don't even render on ios. 
Also, I can use dev tools when I debug on a windows machine to inspect the DOM, is there anyway to achieve the same on Mac?

**Reply by GolfJimB** [reply]
Figured out that removing any external style sheets fixed it!

**Reply by BlaineM-SeriouslyRAD** [reply]
I'm experiencing this with the default (unmodified) .NET MAUI Blazor App template which doesn't contain any external style sheets 🤷‍♀️

**Reply by BlaineM-SeriouslyRAD** [reply]
You can use Safari Technology Preview (https://developer.apple.com/safari/technology-preview/) on the Mac to inspect the DOM.

**Reply by PBTests** [reply]
I have created an issue for this bug. 

https://github.com/dotnet/maui/issues/9723

**Reply by BlaineM-SeriouslyRAD** [reply]
@PBTests Thanks, but there is already a comprehensive one submitted - #7468

**Comment by PBTests** [comment]
Oh, submitted 3 months ago. Rather worrying it’s still an issue. Thanks.
